src/src/AdminBundle/Controller/EventController.php

```diff
@@ -12,12 +13,16 @@
 class EventController extends Controller
 {
     /**
-     * @Route("/event/add", name="event_add")
+     * @Route("/event_type/edit/{id}/event/add", name="event_add")
      * @Method({"GET", "POST"})
      */
-    public function createAction(Request $request)
+    public function createAction($id, Request $request)
     {
+        $repository = $this->getDoctrine()->getRepository(EventType::class);
+        $eventType = $repository->findOneBy(['id' => intval($id)]);
```

Comment:
Netreba nahodou overit ci dany `eventType` existuje? Ak neexsituje nepadne to na nejakej hnusnej databazovej chybe?